agent/engine/task_manager_linux_test.go

```diff
@@ -0,0 +1,69 @@
+// +build linux
+
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package engine
+
+import (
+	"testing"
+
+	"github.com/aws/amazon-ecs-agent/agent/api"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestSetupCgroupWithInvalidSpec checks if managedTask can be setup with an
+// invalid cgroup spec
+func TestSetupCgroupWithInvalidSpec(t *testing.T) {
+	t.Skip()
```

Comment:
Yes, we plan to have a cgroupSpec builder in the following PR. 
I can get rid of this in this PR if its bothering you